ocaml/typing/env.mli

```diff
@@ -59,8 +59,13 @@ type address =
 type t
 
 val empty: t
-val initial_safe_string: t
-val initial_unsafe_string: t
+
+(* Lazy: these should be forced only after command-line flags are parsed, as
```

Comment:
Echo my comment above. I think these comments should explain that the env is what is implied by the extensions in force at the time of forcing. That is, enabling extensions later won't seg-fault, but the env won't be updated.

Reply:
Adjusted - see if you like it.